packages/backend/src/Controllers/Roll/getElectionRollController.ts

```diff
@@ -40,7 +47,15 @@ const getByVoterID = async (req: IElectionRequest, res: Response, next: NextFunc
         Logger.info(req, msg);
         throw new BadRequest(msg)
     }
-    res.json({ electionRollEntry: electionRollEntry })
+
+    // Scrub ballot_id to prevent linking voters to ballots
+    const scrubbedEntry = {
```

Comment:
This looks  safe too. I don't think the code path is actually used at the moment, since the viewing an individual roll is currently passed an entry from the full list

https://github.com/Equal-Vote/bettervoting/blob/409273b8f44e29e755d8c753cd292ce8d1c918ca/packages/frontend/src/components/Election/Admin/EditElectionRoll.tsx#L22